services/apps/src/apps/persistence/categories.clj

```diff
@@ -0,0 +1,18 @@
+(ns apps.persistence.categories
+  (:use [kameleon.queries :only [get-user-id]]
+        [korma.core :exclude [update]]
+        [korma.db :only [transaction]]))
+
+(defn add-hierarchy-version
+  [username version]
+  (insert :app_hierarchy_version
+          (values {:version version
+                   :applied_by (get-user-id username)})))
+
+(defn get-active-hierarchy-version
+  []
+  ((comp :version first)
+   (select :app_hierarchy_version
+           (fields :version)
+           (order :applied :DESC)
+           (limit 1))))
```

Comment:
The last ontology version `applied` is considered the active version (last applied wins).